<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>jstat_profiler_data_visualizer</class>
 <widget class="QWidget" name="jstat_profiler_data_visualizer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>560</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1320</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Jstat Java Virtual Machine Profiling Graphs</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>1318</width>
     <height>570</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>1318</width>
     <height>570</height>
    </size>
   </property>
   <property name="title">
    <string>Jstat Java Profile information</string>
   </property>
   <widget class="QCustomPlot" name="graph_class" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>421</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="graph_gc" native="true">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>421</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="graph_compiler" native="true">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>50</y>
      <width>421</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>536</x>
      <y>30</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Overall Heap Usage Behaviour</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of classes loaded per unit time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>30</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of compilationn tasks per unit time</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>520</y>
      <width>411</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>0</y>
       <width>181</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>default configuration</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>181</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>tuned configuration</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
